src/index.ts

```diff
@@ -2,3 +2,4 @@
  * @since 1.0.0
  */
 export * as Hello from "./Hello.js"
+export * as sha256 from "./sha256.js"
```

Comment:
Let's name modules `Sha256`, etc. to follow the naming patterns of the effect ecosystem.